src/main/java/org/cryptomator/windows/common/NativeLibLoader.java

```diff
@@ -12,7 +12,9 @@
 public class NativeLibLoader {
 
 	private static final Logger LOG = LoggerFactory.getLogger(NativeLibLoader.class);
-	private static final String LIB = "/integrations.dll";
+	private static final String X64_LIB = "/integrations-x64.dll";
+	private static final String ARM64_LIB = "/integrations-arm64.dll";
+	private static String LIBNAME;
```

Comment:
LIBNAME variable is only used inside the only static method. `loadLib()`. And the result is not reused. Hence, removed it from the class and onyl define inside the method.

Reply:
@purejava reminder

Reply:
@infeo Done with d776bcb38a9fa48304c7554284bcc32db6e4696b.